src/docs/sphinx/theory_reference/solid.rst

```diff
@@ -0,0 +1,199 @@
+then use the conservation of linear momentum to formulate the boundary
+value problem
+
+.. math::
+
+   \begin{align*}
+   \nabla_\mathbf{x} \cdot \mathbf{\sigma}(\mathbf{F}) + \rho \mathbf{b} &= \rho \ddot{\mathbf{u}} \\
+   \mathbf{u} &= \mathbf{u}_D & \text{on } \Gamma_D \\
+   \sigma \mathbf{n} &= \mathbf{t} & \text{on } \Gamma_N \\
+   \mathbf{u}(\mathbf{x}, 0) & = \mathbf{u}_0 \\
+   \dot{\mathbf{u}}(\mathbf{x},0) & = \dot{\mathbf{u}}_0
+   \end{align*}
+
+where
+
+.. math::
+
+   \begin{align*}
+   \sigma(\mathbf{F}) &= \text{Cauchy stress via constitutive response} \\
+   \rho &= \text{density} \\
+   \mathbf{b} &= \text{body force} \\
+   \mathbf{u}_D & = \text{fixed boundary} \\
+   \mathbf{t} &= \text{boundary traction} \\
+   \mathbf{u}_0 &= \text{initial displacement} \\
+   \dot{\mathbf{u}}_0 &= \text{initial velocity}
+   \end{align*}
+
+and :math:`\nabla_\mathbf{x}` implies the gradient with respect to the
+current (deformed) configuration. 
+
+Weak Form
+=========
+
+Multiplying the PDE by a vector-valued
+test function :math:`\delta \mathbf{v}` and integrating by parts yields
+the weak form
+
+.. math::
+
+   \begin{align*}
+   &\text{Find } \mathbf{u} \in \mathbf{U} \text{ such that}\\
+   &\int_{\Omega_t} \left( \sigma(\mathbf{u}) \cdot \nabla_\mathbf{x} \delta \mathbf{v} - \rho \mathbf{b} \cdot \delta \mathbf{v}\right) dV - \int_{\Gamma_{N_t}} \delta\mathbf{v}\cdot \mathbf{t}\, dA + \int_{\Omega_t} \rho\ddot{\mathbf{u}} \cdot \delta\mathbf{v} \,dV = 0, & & \forall \delta\mathbf{v} \in \hat{\mathbf{U}}
+   \end{align*}
+
+where
+
+.. math::
+
+   \begin{align*}
+   \mathbf{U} &= \left\{ \mathbf{u} \in H_1^\text{dim}(\Omega):\mathbf{u}=\mathbf{u}_D \text{ on } \Gamma_D \right\} \\
+   \hat{\mathbf{U}} &= \left\{\mathbf{u} \in H_1^\text{dim}(\Omega):\mathbf{u}=\mathbf{0} \text{ on } \Gamma_D \right\}.
+   \end{align*}
+
+and :math:`\Omega` is the current (deformed) configuration. In
+mechanics, the weak form is often referred to as the *principle of
+virtual power*. As Serac uses hyperelastic models, it is convenient to
+write this equation in the reference (undeformed) configuration
+
+.. math::
+
+   \begin{align*}
+   &\int_{\Omega_0} \sigma(\mathbf{u}) \cdot \left(\nabla_\mathbf{X} \delta \mathbf{v} \mathbf{F}^{-1} \right) \text{det}\mathbf{F}\, dV_0  - \int_{\Omega_0} \rho_0 \mathbf{b} \cdot \delta \mathbf{v} dV_0 \\ 
+   & \;\;\;\;\;\;\;\;\;\;\;\; - \int_{\Gamma_{N_0}} \delta\mathbf{v}\cdot \mathbf{t}\,||\mathbf{F}^{-T}\mathbf{n}_0||\text{det}\mathbf{F}\, dA_0 + \int_{\Omega_0} \rho_0\ddot{\mathbf{u}} \cdot \delta\mathbf{v}\, dV_0= 0, & & \forall \delta\mathbf{v} \in \hat{\mathbf{U}}
+   \end{align*}
+
+where :math:`\nabla_X` is the gradient with respect to the reference (material) coordinates.
+
+Material Models
+===============
+
+Serac uses *hyperelastic* material formulations, i.e. materials that behave elastically under large deformations. Mathematically,
```

Comment:
Serac currently is restricted to *hyperelastic* material formulations, i.e. materials that behave in a reversibly elastic fashion under large deformations.